liquidator/detail/score_matrix_detail.h

```diff
@@ -79,10 +79,10 @@ scale(const PWM& pwm, const std::pair<double, double>& min_max, const unsigned r
 }
 
 // returns score; sequences with invalid characters return 0.
-unsigned score(const std::vector<std::array<unsigned, AlphabetSize>>& matrix,
-               const std::string& sequence,
-               const size_t begin,
-               const size_t end)
+inline unsigned score(const std::vector<std::array<unsigned, AlphabetSize>>& matrix,
```

Comment:
Not a big deal if it's working, but my understanding is that inline is only ever a hint, so this may not be portable. Making them static inline should guarantee it builds everywhere.